PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -1 +1,269 @@
+2. emission intensity data is available at the hourly (or sub-hourly) level for the national or local grid.
+
+In cases where this methodology is used to generate carbon credits, extra requirements shall be met:
+
+3. PV arrays are not installed on more than 10% of all buildings in the market of operation
+
+
+## Project boundary
+
+GHG sources included or excluded from the project boundary.
+
+| Source                     | Gas     | Included | Justification           |
+| -------------------------- |---------|----------|------------------------ |
+| Grid electricity displaced | $CO_2$  | Yes      | Main emission source    |
+| Grid electricity displaced | $NO_x$, $SO_x$  | No       | Emissions data is not sufficiently granular and available the half hour or hour |
+
+
+## Measurement
+
+This section introduces the underlying data requirements.
+
+### Asset data
+
+PV production data shall be acquired via automatic digital communication to PeerCo's cloud once the project is established. However, in the initial phase of setting up a project (qualifying the site, its data and quantifying the benefits of the scheme), offline extractions of the data in an open format (which may be CSV, XLSX, etc.) may be accepted.
+
+All self-consumption data (and related generation data) shall be quality controlled prior its usage to generate verified certificates. This verification shall include:
+1. the maximum PV generation is in line with the PV asset's nominal rating,
+2. self-consumption is always lower than the total consumption,
+3. no significant PV generation is happening when there is no solar irradiance (e.g. at night).
+
+Where data is missing, filling of the gaps sall be allowed under the following conditions:
+1. data is not missing more than 5% of the time,
+2. gaps are filled with data whose daily profile and magnitude is consistent with the rest of the asset generation,
+3. if accumulated values are available for the relevant energy measurements, the sum of the generation over the missing data periods shall equate the generation indicated in the accumulated measurements.
+
+When carrying out this asset data verification, a tolerance should be given for numerical purposes (e.g. to account for rounding of measurements by digital systems). This tolerance shall however not be of a magnitude that significantly impacts the volume of the certificates generated.
+
+### Emission intensity data
+
+The emission intensity data used in calculations shall originate from the source in the table below for the geography corresponding to the location of the assets.
+
+| Geography     | Emission intensity source                              |
+|---------------|--------------------------------------------------------|
+| Great Britain | [National Grid ESO's *carbonintensity.org.uk* API](https://www.carbonintensity.org.uk/) |
+| Germany, Nigeria, Portugal, Spain, South-Africa, Switzerland | [ElectricityMaps API](https://www.electricitymaps.com/) |
```

Comment:
We can add the EIA for the US

Reply:
Maybe we should instead require a disclosure of where the emissions data is coming from as part of the EAC - with a criterion that it should be at least hourly and correspond to an average intensity signal.

I agree that this would make the method easier to scale to new countries without amending.

I'll propose an update of this in a new commit.

@mattwc-openeac - happy to add the EIA if you provide me the link to that dataset.

Reply:
agree